src/resources/language/_language-tr.yml

```diff
@@ -1,16 +1,16 @@
-toc-title-document: "Içindekiler"
+toc-title-document: "İçindekiler"
 toc-title-website: "Bu sayfada"
 
 related-formats-title: "Diğer Formatlar"
-related-notebooks-title: "Notebooks"
+related-notebooks-title: "Notebooklar"
 source-notebooks-prefix: "Kaynak"
 other-links-title: "Diğer Bağlantılar"
 code-links-title: "Kod Bağlantıları"
-launch-dev-container-title: "Dev Container Başlat"
+launch-dev-container-title: "Geliştirme Konteynerını Başlat"
```

Comment:
"Dev Container" is a [term of art](https://containers.dev/) - should we be translating that?

Reply:
thank you for the flag.